rwasm/runtime/wasmtime/builder.go

```diff
@@ -0,0 +1,93 @@
+package runtimewasmtime
+
+import (
+	"github.com/bytecodealliance/wasmtime-go"
+	"github.com/pkg/errors"
+	"github.com/suborbital/reactr/rwasm/moduleref"
+	"github.com/suborbital/reactr/rwasm/runtime"
+)
+
+// WasmTimeBuilder is a Wasmer implementation of the instanceBuilder interface
+type WasmTimeBuilder struct {
+	ref     *moduleref.WasmModuleRef
+	hostFns []runtime.HostFn
+	module  *wasmtime.Module
+	engine  *wasmtime.Engine
+	linker  *wasmtime.Linker
+}
+
+// NewBuilder creates a new WasmTimeBuilder
+func NewBuilder(ref *moduleref.WasmModuleRef, hostFns ...runtime.HostFn) runtime.RuntimeBuilder {
+	w := &WasmTimeBuilder{
+		ref:     ref,
+		hostFns: hostFns,
+	}
+
+	return w
+}
+
+func (w *WasmTimeBuilder) New() (runtime.RuntimeInstance, error) {
+	module, engine, linker, err := w.internals()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to internals")
+	}
+
+	store := wasmtime.NewStore(engine)
+
+	wasiConfig := wasmtime.NewWasiConfig()
+	store.SetWasi(wasiConfig)
+
+	wasmTimeInst, err := linker.Instantiate(store, module)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to linker.Instantiate")
+	}
+
+	inst := &WasmtimeInstance{
+		inst:  *wasmTimeInst,
+		store: store,
+	}
+
+	if _, err := inst.Call("_start"); err != nil {
+		if errors.Is(err, runtime.ErrExportNotFound) {
+			// that's ok, not all modules will have _start
+		} else {
+			return nil, errors.Wrap(err, "failed to call _start")
```

Comment:
Would be helpful if this said something that made it obvious that the `_start` function was defined, just that there was an error executing it. I totally see myself seeing this error and going "well does my module even have a _start??"

Maybe something like this:
```suggestion
			return nil, errors.Wrap(err, "failed to call defined _start function")
```
but I imagine there's a better way to word it.